requirements/_release_tools.txt

```diff
@@ -1 +1 @@
-changelist==0.1
```

Comment:
I assume you are anticipating a release that's not out yet.

Reply:
https://pypi.org/project/changelist/0.4/